custom/icds_reports/utils/fix_past_data_due_to_change_in_height_eligibilty.sql

```diff
@@ -0,0 +1,93 @@
+-- child health table fix
+UPDATE child_health_monthly
+   SET
+      height_measured_in_month = tmp.height_measured_in_month, 
+      current_month_stunting = tmp.current_month_stunting,
+      stunting_last_recorded = tmp.stunting_last_recorded,
+      wasting_last_recorded = tmp.wasting_last_recorded,
+      current_month_wasting = tmp.current_month_wasting
+   FROM (
+      SELECT
+         chm.case_id as case_id,
+         chm.valid_in_month as valid_in_month,
+         chm.age_tranche as age_tranche,
+         CASE
+            WHEN NOT (chm.valid_in_month::boolean AND chm.age_tranche::Integer <= 60) THEN NULL
+         ELSE
+            chm.current_month_stunting
+         END as current_month_stunting,
+         CASE
+            WHEN NOT (chm.valid_in_month::boolean AND chm.age_tranche::Integer <= 60) THEN NULL
+         ELSE
+            chm.stunting_last_recorded
+         END as stunting_last_recorded,
+         CASE
+            WHEN NOT (chm.valid_in_month::boolean AND chm.age_tranche::Integer <= 60) THEN NULL
+         ELSE
+            chm.wasting_last_recorded
+         END as wasting_last_recorded,
+         CASE
+            WHEN NOT (chm.valid_in_month::boolean AND chm.age_tranche::Integer <= 60) THEN NULL
+         ELSE
+            chm.current_month_wasting
+         END as current_month_wasting,
+         CASE
+            WHEN date_trunc('MONTH', gm.height_child_last_recorded) = '2018-06-01' AND (chm.valid_in_month::boolean AND chm.age_tranche::Integer <= 60) THEN 1
+         ELSE
+            0
+         END as height_measured_in_month
+         FROM child_health_monthly chm
```

Comment:
you dont need a join here. you can make it same as you updated in your another Pr